make explicit the location of mutate (#354)
vercel/swr-site: pages/docs/mutation.en-US.md

## mutate

```js

> import { mutate, useSWRConfig } from "swr"

Reviewer: We should promote the `mutate` from `useSWRConfig` IMO to make sure `mutate` is really from the current context, bounded with the cache from that context.

```suggestion
import { mutate as globalMutate, useSWRConfig } from "swr"
```

function App() {
  // Or from the useSWRConfig hook

>   // const { mutate } = useSWRConfig()
>   const data = await mutate(key, data, options)

Reviewer: ```suggestion
  const { mutate } = useSWRConfig()
  const data = mutate(key, data, options)
  
  // Or you can use globalMutate directly
  // await globalMutate(key, data, options)
  
```

}
```

> You can get [bound mutate](/docs/mutation#bound-mutate) from `useSWR`.

Reviewer: ```suggestion
[Bound Mutate](/docs/mutation#bound-mutate) is the short path to mutate the current key with data

#### Example
```
